sos/report/plugins/microk8s.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2023 Canonical Ltd.,
+#                    David Negreira <david.negreira@canonical.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, UbuntuPlugin
+
+
+class Microk8s(Plugin, UbuntuPlugin):
+
+    short_desc = 'The lightweight Kubernetes'
+    plugin_name = "microk8s"
+    profiles = ('container',)
+
+    packages = ('microk8s')
```

Comment:
You need to have

    packages = ('microk8s',)

here to force python a tuple and not list. Since:

    >>> ('microk8s').__class__.__name__
    'str'
    >>> 

(that is the reason for test failure:

    tests/unittests/conformance_tests.py:PluginConformance.test_plugin_tuples_set_correctly:  FAIL: AssertionError: 'microk8s' is not an instance of <class 'tuple'> : Microk8s.packages is type <class 'str'> (0.51 s)

Reply:
Whoops. Fixed on https://github.com/sosreport/sos/commit/e5a6728e7fb10d308a0b916dba83842c9be76a29